docs/Deploy/Deploy-Fleet-on-Kubernetes.md

```diff
@@ -154,14 +83,14 @@ Note: this step is not neccessary when using the Fleet Helm Chart as it handles
 The last step is to run the Fleet database migrations on your new MySQL server. To do this, run the following:
 
 ```sh
-kubectl create -f ./docs/Using-Fleet/configuration-files/kubernetes/fleet-migrations.yml
+kubectl create -f ./docs/Deploy/kubernetes/fleet-migrations.yml
```

Comment:
I'm not seeing a `/kubernetes/` directory, so I think all of these URLs are broken: https://github.com/fleetdm/fleet/blob/main/docs/Deploy/kubernetes

Reply:
> I'm not seeing a /kubernetes/ directory

I think it's getting added as part of this PR? 
![Screenshot 2024-02-22 at 4 30 46 PM](https://github.com/fleetdm/fleet/assets/47070608/7c7da069-c389-44de-9b24-d6f0619e8fa4)


(I could be missing something)

Reply:
🤦 Nope, that's me. My mistake.